example-monorepo/packages/plugin-test/.env.op

```diff
@@ -0,0 +1,32 @@
+# for now we'll import directly via local paths in node_modules
+# but soon we'll add ways of importing using package name and likely relying on package.json
+# # @plugin(./node_modules/@varlock/simple-vault-plugin/dist/plugin.js)
+# # @plugin(node_modules:@varlock/simple-vault-plugin)
+# # @plugin(npm:@varlock/simple-vault-plugin@1.2.3)
+#
+# # @initSimpleVault(id=vault1, key=SIMPLE_VAULT_ENCRYPTION_KEY)
+#
+# @plugin(./node_modules/@varlock/1password-plugin/dist/plugin.js)
+# # @plugin(https://varlock-plugins-registry.dmno.workers.dev/varlock-1password-plugin-0.0.0.tgz)
+# @initOpVault(id=v1, token=$OP_TOKEN, enableCliAuth=true)
```

Comment:
most plugins will introduce a root decorator that will be responsible for initialization - which usually means setting options, and wiring up auth to env items. In many cases you may want to create multiple instances of a plugin like this.